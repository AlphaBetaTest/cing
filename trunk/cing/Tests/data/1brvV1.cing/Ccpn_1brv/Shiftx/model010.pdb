REMARK PDB file of molecule Ccpn_1brv                                 
ATOM      1  N   VAL A 171       4.467   1.669   4.045  1.00  0.00              
ATOM      2  CA  VAL A 171       4.394   0.611   2.987  1.00  0.00              
ATOM      3  HA  VAL A 171       4.341   1.206   2.080  1.00  0.00              
ATOM      4  CB  VAL A 171       5.751  -0.174   2.983  1.00  0.00              
ATOM      5  HB  VAL A 171       6.541   0.595   3.030  1.00  0.00              
ATOM      6  CG1 VAL A 171       5.916  -1.146   4.172  1.00  0.00              
ATOM      7 HG11 VAL A 171       5.769  -0.627   5.138  1.00  0.00              
ATOM      8 HG12 VAL A 171       5.179  -1.970   4.133  1.00  0.00              
ATOM      9 HG13 VAL A 171       6.924  -1.598   4.202  1.00  0.00              
ATOM     10  CG2 VAL A 171       6.102  -0.921   1.681  1.00  0.00              
ATOM     11 HG21 VAL A 171       5.332  -1.644   1.373  1.00  0.00              
ATOM     12 HG22 VAL A 171       6.225  -0.222   0.839  1.00  0.00              
ATOM     13 HG23 VAL A 171       7.058  -1.469   1.751  1.00  0.00              
ATOM     14  C   VAL A 171       3.097  -0.283   2.811  1.00  0.00              
ATOM     15  O   VAL A 171       3.010  -0.936   1.769  1.00  0.00              
ATOM     16  N   PRO A 172       2.070  -0.387   3.695  1.00  0.00              
ATOM     17  CD  PRO A 172       2.190   0.033   5.091  1.00  0.00              
ATOM     18  CA  PRO A 172       0.815  -1.123   3.392  1.00  0.00              
ATOM     19  HA  PRO A 172       0.990  -1.971   2.697  1.00  0.00              
ATOM     20  CB  PRO A 172       0.482  -1.705   4.780  1.00  0.00              
ATOM     21  HB2 PRO A 172      -0.604  -1.853   4.935  1.00  0.00              
ATOM     22  HB3 PRO A 172       0.942  -2.709   4.871  1.00  0.00              
ATOM     23  CG  PRO A 172       1.106  -0.764   5.823  1.00  0.00              
ATOM     24  HG2 PRO A 172       0.349  -0.080   6.240  1.00  0.00              
ATOM     25  HG3 PRO A 172       1.519  -1.324   6.683  1.00  0.00              
ATOM     26  HD2 PRO A 172       2.044   1.131   5.142  1.00  0.00              
ATOM     27  HD3 PRO A 172       3.207  -0.190   5.471  1.00  0.00              
ATOM     28  C   PRO A 172      -0.256  -0.157   2.772  1.00  0.00              
ATOM     29  O   PRO A 172      -0.831   0.681   3.478  1.00  0.00              
ATOM     30  N   CYS A 173      -0.540  -0.268   1.457  1.00  0.00              
ATOM     31  H   CYS A 173      -0.017  -0.969   0.926  1.00  0.00              
ATOM     32  CA  CYS A 173      -1.554   0.572   0.748  1.00  0.00              
ATOM     33  HA  CYS A 173      -1.224   1.613   0.778  1.00  0.00              
ATOM     34  CB  CYS A 173      -1.474   0.238  -0.743  1.00  0.00              
ATOM     35  HB2 CYS A 173      -0.875  -0.649  -0.971  1.00  0.00              
ATOM     36  HB3 CYS A 173      -2.475   0.087  -1.126  1.00  0.00              
ATOM     37  SG  CYS A 173      -0.657   1.548  -1.644  1.00  0.00              
ATOM     38  C   CYS A 173      -2.999   0.631   1.354  1.00  0.00              
ATOM     39  O   CYS A 173      -3.623   1.697   1.322  1.00  0.00              
ATOM     40  N   SER A 174      -3.489  -0.467   1.961  1.00  0.00              
ATOM     41  H   SER A 174      -2.867  -1.280   1.893  1.00  0.00              
ATOM     42  CA  SER A 174      -4.729  -0.449   2.806  1.00  0.00              
ATOM     43  HA  SER A 174      -5.569  -0.156   2.173  1.00  0.00              
ATOM     44  CB  SER A 174      -4.976  -1.885   3.320  1.00  0.00              
ATOM     45  HB2 SER A 174      -5.949  -1.921   3.852  1.00  0.00              
ATOM     46  HB3 SER A 174      -5.077  -2.582   2.470  1.00  0.00              
ATOM     47  OG  SER A 174      -3.941  -2.341   4.201  1.00  0.00              
ATOM     48  HG  SER A 174      -4.010  -1.822   5.010  1.00  0.00              
ATOM     49  C   SER A 174      -4.802   0.555   4.010  1.00  0.00              
ATOM     50  O   SER A 174      -5.872   0.784   4.577  1.00  0.00              
ATOM     51  N   THR A 175      -3.656   1.111   4.403  1.00  0.00              
ATOM     52  H   THR A 175      -2.849   0.783   3.857  1.00  0.00              
ATOM     53  CA  THR A 175      -3.522   2.132   5.478  1.00  0.00              
ATOM     54  HA  THR A 175      -4.476   2.168   6.038  1.00  0.00              
ATOM     55  CB  THR A 175      -2.426   1.672   6.490  1.00  0.00              
ATOM     56  HB  THR A 175      -2.492   2.335   7.377  1.00  0.00              
ATOM     57  OG1 THR A 175      -1.119   1.813   5.940  1.00  0.00              
ATOM     58  HG1 THR A 175      -1.113   1.356   5.085  1.00  0.00              
ATOM     59  CG2 THR A 175      -2.571   0.214   6.970  1.00  0.00              
ATOM     60 HG21 THR A 175      -3.588   0.023   7.362  1.00  0.00              
ATOM     61 HG22 THR A 175      -2.420  -0.499   6.134  1.00  0.00              
ATOM     62 HG23 THR A 175      -1.837  -0.040   7.749  1.00  0.00              
ATOM     63  C   THR A 175      -3.270   3.590   4.941  1.00  0.00              
ATOM     64  O   THR A 175      -3.578   4.555   5.642  1.00  0.00              
ATOM     65  N   CYS A 176      -2.708   3.764   3.721  1.00  0.00              
ATOM     66  H   CYS A 176      -2.770   2.943   3.114  1.00  0.00              
ATOM     67  CA  CYS A 176      -2.542   5.079   3.058  1.00  0.00              
ATOM     68  HA  CYS A 176      -2.067   5.791   3.762  1.00  0.00              
ATOM     69  CB  CYS A 176      -1.546   4.758   1.921  1.00  0.00              
ATOM     70  HB2 CYS A 176      -1.908   4.044   1.187  1.00  0.00              
ATOM     71  HB3 CYS A 176      -1.334   5.614   1.262  1.00  0.00              
ATOM     72  SG  CYS A 176       0.013   4.176   2.616  1.00  0.00              
ATOM     73  C   CYS A 176      -3.907   5.717   2.619  1.00  0.00              
ATOM     74  O   CYS A 176      -4.114   6.917   2.803  1.00  0.00              
ATOM     75  N   GLU A 177      -4.811   4.910   2.029  1.00  0.00              
ATOM     76  H   GLU A 177      -4.428   4.010   1.731  1.00  0.00              
ATOM     77  CA  GLU A 177      -6.227   5.261   1.723  1.00  0.00              
ATOM     78  HA  GLU A 177      -6.527   4.493   0.996  1.00  0.00              
ATOM     79  CB  GLU A 177      -7.100   5.064   3.005  1.00  0.00              
ATOM     80  HB2 GLU A 177      -6.736   5.759   3.787  1.00  0.00              
ATOM     81  HB3 GLU A 177      -8.134   5.406   2.802  1.00  0.00              
ATOM     82  CG  GLU A 177      -7.184   3.633   3.600  1.00  0.00              
ATOM     83  HG2 GLU A 177      -6.174   3.279   3.874  1.00  0.00              
ATOM     84  HG3 GLU A 177      -7.737   3.659   4.557  1.00  0.00              
ATOM     85  CD  GLU A 177      -7.826   2.584   2.693  1.00  0.00              
ATOM     86  OE1 GLU A 177      -7.225   1.976   1.814  1.00  0.00              
ATOM     87  OE2 GLU A 177      -9.148   2.410   2.958  1.00  0.00              
ATOM     88  C   GLU A 177      -6.520   6.592   0.972  1.00  0.00              
ATOM     89  O   GLU A 177      -7.394   7.387   1.328  1.00  0.00              
ATOM     90  N   GLY A 178      -5.793   6.772  -0.130  1.00  0.00              
ATOM     91  H   GLY A 178      -4.903   6.270  -0.062  1.00  0.00              
ATOM     92  CA  GLY A 178      -5.874   8.005  -0.936  1.00  0.00              
ATOM     93  HA2 GLY A 178      -5.880   7.679  -1.985  1.00  0.00              
ATOM     94  HA3 GLY A 178      -6.834   8.508  -0.776  1.00  0.00              
ATOM     95  C   GLY A 178      -4.744   9.046  -0.776  1.00  0.00              
ATOM     96  O   GLY A 178      -4.760  10.055  -1.481  1.00  0.00              
ATOM     97  N   ASN A 179      -3.753   8.814   0.106  1.00  0.00              
ATOM     98  H   ASN A 179      -3.692   7.836   0.408  1.00  0.00              
ATOM     99  CA  ASN A 179      -2.574   9.687   0.231  1.00  0.00              
ATOM    100  HA  ASN A 179      -2.817  10.719  -0.090  1.00  0.00              
ATOM    101  CB  ASN A 179      -1.975   9.576   1.646  1.00  0.00              
ATOM    102  HB2 ASN A 179      -2.744   9.403   2.425  1.00  0.00              
ATOM    103  HB3 ASN A 179      -1.331   8.673   1.616  1.00  0.00              
ATOM    104  CG  ASN A 179      -1.074  10.743   2.012  1.00  0.00              
ATOM    105  OD1 ASN A 179       0.089  10.807   1.632  1.00  0.00              
ATOM    106  ND2 ASN A 179      -1.588  11.708   2.719  1.00  0.00              
ATOM    107 HD21 ASN A 179      -0.938  12.471   2.918  1.00  0.00              
ATOM    108 HD22 ASN A 179      -2.570  11.602   2.987  1.00  0.00              
ATOM    109  C   ASN A 179      -1.559   9.154  -0.766  1.00  0.00              
ATOM    110  O   ASN A 179      -0.915   8.116  -0.572  1.00  0.00              
ATOM    111  N   LEU A 180      -1.406   9.893  -1.840  1.00  0.00              
ATOM    112  H   LEU A 180      -2.027  10.701  -1.928  1.00  0.00              
ATOM    113  CA  LEU A 180      -0.589   9.415  -2.948  1.00  0.00              
ATOM    114  HA  LEU A 180      -0.998   8.397  -3.035  1.00  0.00              
ATOM    115  CB  LEU A 180      -0.980  10.349  -4.103  1.00  0.00              
ATOM    116  HB2 LEU A 180      -1.148  11.379  -3.711  1.00  0.00              
ATOM    117  HB3 LEU A 180      -0.123  10.398  -4.743  1.00  0.00              
ATOM    118  CG  LEU A 180      -2.157   9.913  -4.985  1.00  0.00              
ATOM    119  HG  LEU A 180      -2.948   9.702  -4.269  1.00  0.00              
ATOM    120  CD1 LEU A 180      -2.614  11.049  -5.913  1.00  0.00              
ATOM    121 HD11 LEU A 180      -2.888  11.959  -5.348  1.00  0.00              
ATOM    122 HD12 LEU A 180      -1.828  11.337  -6.637  1.00  0.00              
ATOM    123 HD13 LEU A 180      -3.507  10.763  -6.499  1.00  0.00              
ATOM    124  CD2 LEU A 180      -1.865   8.653  -5.821  1.00  0.00              
ATOM    125 HD21 LEU A 180      -1.585   7.787  -5.194  1.00  0.00              
ATOM    126 HD22 LEU A 180      -2.746   8.341  -6.412  1.00  0.00              
ATOM    127 HD23 LEU A 180      -1.036   8.814  -6.536  1.00  0.00              
ATOM    128  C   LEU A 180       0.940   9.204  -2.721  1.00  0.00              
ATOM    129  O   LEU A 180       1.563   8.395  -3.400  1.00  0.00              
ATOM    130  N   ALA A 181       1.498   9.902  -1.735  1.00  0.00              
ATOM    131  H   ALA A 181       0.773  10.457  -1.283  1.00  0.00              
ATOM    132  CA  ALA A 181       2.852   9.633  -1.204  1.00  0.00              
ATOM    133  HA  ALA A 181       3.537   9.554  -2.062  1.00  0.00              
ATOM    134  CB  ALA A 181       3.260  10.853  -0.377  1.00  0.00              
ATOM    135  HB1 ALA A 181       3.243  11.775  -0.984  1.00  0.00              
ATOM    136  HB2 ALA A 181       2.590  10.992   0.493  1.00  0.00              
ATOM    137  HB3 ALA A 181       4.287  10.722   0.010  1.00  0.00              
ATOM    138  C   ALA A 181       2.925   8.290  -0.393  1.00  0.00              
ATOM    139  O   ALA A 181       3.755   7.442  -0.711  1.00  0.00              
ATOM    140  N   CYS A 182       2.036   8.076   0.604  1.00  0.00              
ATOM    141  H   CYS A 182       1.225   8.703   0.538  1.00  0.00              
ATOM    142  CA  CYS A 182       1.888   6.775   1.332  1.00  0.00              
ATOM    143  HA  CYS A 182       2.738   6.526   1.930  1.00  0.00              
ATOM    144  CB  CYS A 182       0.828   7.054   2.392  1.00  0.00              
ATOM    145  HB2 CYS A 182       0.981   8.022   2.879  1.00  0.00              
ATOM    146  HB3 CYS A 182      -0.150   7.128   1.922  1.00  0.00              
ATOM    147  SG  CYS A 182       0.746   5.733   3.608  1.00  0.00              
ATOM    148  C   CYS A 182       1.664   5.532   0.417  1.00  0.00              
ATOM    149  O   CYS A 182       2.333   4.501   0.518  1.00  0.00              
ATOM    150  N   LEU A 183       0.721   5.697  -0.501  1.00  0.00              
ATOM    151  H   LEU A 183       0.212   6.589  -0.425  1.00  0.00              
ATOM    152  CA  LEU A 183       0.436   4.766  -1.598  1.00  0.00              
ATOM    153  HA  LEU A 183       0.171   3.787  -1.165  1.00  0.00              
ATOM    154  CB  LEU A 183      -0.769   5.456  -2.271  1.00  0.00              
ATOM    155  HB2 LEU A 183      -0.484   6.485  -2.542  1.00  0.00              
ATOM    156  HB3 LEU A 183      -0.914   5.011  -3.212  1.00  0.00              
ATOM    157  CG  LEU A 183      -2.163   5.434  -1.641  1.00  0.00              
ATOM    158  HG  LEU A 183      -2.018   5.943  -0.675  1.00  0.00              
ATOM    159  CD1 LEU A 183      -3.128   6.208  -2.556  1.00  0.00              
ATOM    160 HD11 LEU A 183      -3.093   5.840  -3.600  1.00  0.00              
ATOM    161 HD12 LEU A 183      -4.176   6.113  -2.235  1.00  0.00              
ATOM    162 HD13 LEU A 183      -2.893   7.285  -2.592  1.00  0.00              
ATOM    163  CD2 LEU A 183      -2.725   4.013  -1.469  1.00  0.00              
ATOM    164 HD21 LEU A 183      -2.804   3.478  -2.436  1.00  0.00              
ATOM    165 HD22 LEU A 183      -2.086   3.394  -0.824  1.00  0.00              
ATOM    166 HD23 LEU A 183      -3.728   4.009  -1.007  1.00  0.00              
ATOM    167  C   LEU A 183       1.592   4.537  -2.639  1.00  0.00              
ATOM    168  O   LEU A 183       1.727   3.433  -3.164  1.00  0.00              
ATOM    169  N   SER A 184       2.394   5.566  -2.962  1.00  0.00              
ATOM    170  H   SER A 184       2.175   6.401  -2.399  1.00  0.00              
ATOM    171  CA  SER A 184       3.614   5.409  -3.820  1.00  0.00              
ATOM    172  HA  SER A 184       3.326   4.830  -4.720  1.00  0.00              
ATOM    173  CB  SER A 184       4.091   6.790  -4.320  1.00  0.00              
ATOM    174  HB2 SER A 184       3.265   7.322  -4.830  1.00  0.00              
ATOM    175  HB3 SER A 184       4.390   7.435  -3.470  1.00  0.00              
ATOM    176  OG  SER A 184       5.178   6.655  -5.240  1.00  0.00              
ATOM    177  HG  SER A 184       5.715   5.909  -4.941  1.00  0.00              
ATOM    178  C   SER A 184       4.789   4.601  -3.161  1.00  0.00              
ATOM    179  O   SER A 184       5.597   4.016  -3.886  1.00  0.00              
ATOM    180  N   LEU A 185       4.917   4.603  -1.823  1.00  0.00              
ATOM    181  H   LEU A 185       4.203   5.166  -1.334  1.00  0.00              
ATOM    182  CA  LEU A 185       5.871   3.707  -1.092  1.00  0.00              
ATOM    183  HA  LEU A 185       6.840   3.662  -1.594  1.00  0.00              
ATOM    184  CB  LEU A 185       6.058   4.203   0.377  1.00  0.00              
ATOM    185  HB2 LEU A 185       5.125   3.982   0.933  1.00  0.00              
ATOM    186  HB3 LEU A 185       6.829   3.579   0.871  1.00  0.00              
ATOM    187  CG  LEU A 185       6.397   5.690   0.610  1.00  0.00              
ATOM    188  HG  LEU A 185       5.590   6.259   0.110  1.00  0.00              
ATOM    189  CD1 LEU A 185       6.369   6.032   2.107  1.00  0.00              
ATOM    190 HD11 LEU A 185       5.390   5.792   2.562  1.00  0.00              
ATOM    191 HD12 LEU A 185       7.140   5.480   2.677  1.00  0.00              
ATOM    192 HD13 LEU A 185       6.541   7.110   2.281  1.00  0.00              
ATOM    193  CD2 LEU A 185       7.756   6.090   0.012  1.00  0.00              
ATOM    194 HD21 LEU A 185       7.782   5.926  -1.082  1.00  0.00              
ATOM    195 HD22 LEU A 185       7.974   7.162   0.173  1.00  0.00              
ATOM    196 HD23 LEU A 185       8.591   5.513   0.449  1.00  0.00              
ATOM    197  C   LEU A 185       5.474   2.207  -1.047  1.00  0.00              
ATOM    198  O   LEU A 185       6.345   1.337  -0.974  1.00  0.00              
ATOM    199  N   CYS A 186       4.170   1.923  -1.056  1.00  0.00              
ATOM    200  H   CYS A 186       3.553   2.730  -1.034  1.00  0.00              
ATOM    201  CA  CYS A 186       3.634   0.601  -0.825  1.00  0.00              
ATOM    202  HA  CYS A 186       4.195   0.291   0.069  1.00  0.00              
ATOM    203  CB  CYS A 186       2.143   0.927  -0.516  1.00  0.00              
ATOM    204  HB2 CYS A 186       1.725   0.111   0.072  1.00  0.00              
ATOM    205  HB3 CYS A 186       1.938   1.810   0.103  1.00  0.00              
ATOM    206  SG  CYS A 186       1.195   0.976  -2.045  1.00  0.00              
ATOM    207  C   CYS A 186       3.740  -0.502  -1.928  1.00  0.00              
ATOM    208  O   CYS A 186       4.212  -0.325  -3.054  1.00  0.00              
ATOM    209  N   HIS A 187       3.142  -1.641  -1.570  1.00  0.00              
ATOM    210  H   HIS A 187       3.035  -1.721  -0.553  1.00  0.00              
ATOM    211  CA  HIS A 187       2.916  -2.786  -2.479  1.00  0.00              
ATOM    212  HA  HIS A 187       3.647  -2.757  -3.312  1.00  0.00              
ATOM    213  CB  HIS A 187       3.049  -4.174  -1.782  1.00  0.00              
ATOM    214  HB2 HIS A 187       2.075  -4.658  -1.660  1.00  0.00              
ATOM    215  HB3 HIS A 187       3.569  -4.797  -2.516  1.00  0.00              
ATOM    216  CG  HIS A 187       3.724  -4.313  -0.427  1.00  0.00              
ATOM    217  ND1 HIS A 187       3.093  -3.943   0.746  1.00  0.00              
ATOM    218  CD2 HIS A 187       5.045  -4.680  -0.199  1.00  0.00              
ATOM    219  CE1 HIS A 187       4.146  -4.070   1.602  1.00  0.00              
ATOM    220  NE2 HIS A 187       5.350  -4.545   1.145  1.00  0.00              
ATOM    221  HD2 HIS A 187       5.713  -4.957  -0.998  1.00  0.00              
ATOM    222  HE1 HIS A 187       4.069  -3.558   2.548  1.00  0.00              
ATOM    223  C   HIS A 187       1.458  -2.609  -3.034  1.00  0.00              
ATOM    224  O   HIS A 187       0.468  -2.724  -2.299  1.00  0.00              
ATOM    225  N   ILE A 188       1.321  -2.345  -4.337  1.00  0.00              
ATOM    226  H   ILE A 188       2.213  -2.312  -4.841  1.00  0.00              
ATOM    227  CA  ILE A 188      -0.015  -2.058  -4.962  1.00  0.00              
ATOM    228  HA  ILE A 188      -0.740  -1.743  -4.183  1.00  0.00              
ATOM    229  CB  ILE A 188       0.055  -0.817  -5.926  1.00  0.00              
ATOM    230  HB  ILE A 188       0.733  -1.115  -6.742  1.00  0.00              
ATOM    231  CG2 ILE A 188      -1.331  -0.486  -6.546  1.00  0.00              
ATOM    232 HG21 ILE A 188      -1.764  -1.348  -7.088  1.00  0.00              
ATOM    233 HG22 ILE A 188      -2.068  -0.178  -5.782  1.00  0.00              
ATOM    234 HG23 ILE A 188      -1.270   0.330  -7.288  1.00  0.00              
ATOM    235  CG1 ILE A 188       0.627   0.484  -5.277  1.00  0.00              
ATOM    236 HG12 ILE A 188      -0.099   0.910  -4.558  1.00  0.00              
ATOM    237 HG13 ILE A 188       1.514   0.243  -4.660  1.00  0.00              
ATOM    238  CD1 ILE A 188       1.081   1.576  -6.264  1.00  0.00              
ATOM    239 HD11 ILE A 188       1.828   1.194  -6.984  1.00  0.00              
ATOM    240 HD12 ILE A 188       0.241   2.000  -6.844  1.00  0.00              
ATOM    241 HD13 ILE A 188       1.557   2.419  -5.727  1.00  0.00              
ATOM    242  C   ILE A 188      -0.546  -3.385  -5.617  1.00  0.00              
ATOM    243  O   ILE A 188      -0.394  -3.622  -6.819  1.00  0.00              
ATOM    244  N   GLU A 189      -1.168  -4.256  -4.802  1.00  0.00              
ATOM    245  H   GLU A 189      -1.024  -4.060  -3.811  1.00  0.00              
ATOM    246  CA  GLU A 189      -1.667  -5.579  -5.246  1.00  0.00              
ATOM    247  HA  GLU A 189      -1.039  -5.918  -6.085  1.00  0.00              
ATOM    248  CB  GLU A 189      -1.421  -6.567  -4.073  1.00  0.00              
ATOM    249  HB2 GLU A 189      -0.372  -6.470  -3.725  1.00  0.00              
ATOM    250  HB3 GLU A 189      -2.043  -6.269  -3.203  1.00  0.00              
ATOM    251  CG  GLU A 189      -1.680  -8.060  -4.407  1.00  0.00              
ATOM    252  HG2 GLU A 189      -2.731  -8.216  -4.714  1.00  0.00              
ATOM    253  HG3 GLU A 189      -1.068  -8.370  -5.274  1.00  0.00              
ATOM    254  CD  GLU A 189      -1.382  -9.019  -3.258  1.00  0.00              
ATOM    255  OE1 GLU A 189      -0.465  -9.832  -3.267  1.00  0.00              
ATOM    256  OE2 GLU A 189      -2.253  -8.878  -2.223  1.00  0.00              
ATOM    257  C   GLU A 189      -3.147  -5.521  -5.728  1.00  0.00              
TER     258      GLU A 189 
END   
