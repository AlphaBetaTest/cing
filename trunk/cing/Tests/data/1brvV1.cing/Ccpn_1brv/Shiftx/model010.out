# Entries marked with a * may have inaccurate shift predictions.
# Entries marked with a value -666 should be igonored
# Output format adjusted by GV 11 September 2007
  171   V  N      120.3081
  171   V  CA      60.4367
  171   V  CB      33.7271
  171   V  C      174.5656
  171   V  H        7.8976
  171   V  HA       3.9052
  171   V  HB       1.9612
  171   V  HG11     0.9442
  171   V  HG12     0.9928
  171   V  HG13     0.3408
  171   V  HG21     0.5374
  171   V  HG22     1.0324
  171   V  HG23     0.8255
  172   P  CA      61.1352
  172   P  CB      30.8846
  172   P  C      175.4867
  172   P  HA       4.4571
  172   P  HB2      2.3359
  172   P  HB3      2.2882
  172   P  HD2      3.7169
  172   P  HD3      3.7791
  172   P  HG2      2.0344
  172   P  HG3      2.0640
  173   C  N      123.0185
  173   C  CA      60.0543
  173   C  CB      40.8014
  173   C  C      177.2081
  173   C  H        8.8207
  173   C  HA       4.2127
  173   C  HB2      3.1480
  173   C  HB3      3.2047
  174   S  N      114.3631
  174   S  CA      60.8561
  174   S  CB      62.3887
  174   S  C      175.4369
  174   S  H        8.2227
  174   S  HA       4.1120
  174   S  HB2      4.0209
  174   S  HB3      4.0208
  175   T  N      111.8133
  175   T  CA      62.9896
  175   T  CB      69.8241
  175   T  C      175.8601
  175   T  H        7.7232
  175   T  HA       4.3945
  175   T  HB       4.2436
  175   T  HG21     1.2150
  175   T  HG22     1.2183
  175   T  HG23     1.2163
  176   C  N      119.2814
  176   C  CA      58.6028
  176   C  CB      40.8861
  176   C  C      173.8733
  176   C  H        8.1311
  176   C  HA       4.3469
  176   C  HB2      3.1221
  176   C  HB3      3.2123
  177   E  N      116.8565
  177   E  CA      57.5676
  177   E  CB      27.0138
  177   E  C      175.5014
  177   E  H        8.9735
  177   E  HA       3.7245
  177   E  HB2      2.0957
  177   E  HB3      1.9786
  177   E  HG2      2.3265
  177   E  HG3      2.3784
  178   G  N      104.2347
  178   G  CA      45.1924
  178   G  C      173.4992
  178   G  H        8.3556
  178   G  HA    -666.0000
  178   G  HA2      3.9601
  178   G  HA3      3.9601
  179   N  N      119.2199
  179   N  CA      52.4168
  179   N  CB      39.3151
  179   N  C      175.2769
  179   N  H        7.9486
  179   N  HA       4.6012
  179   N  HB2      2.8025
  179   N  HB3      2.7297
  179   N  HD21     7.1383
  179   N  HD22     6.6732
  180   L  N      129.1122
  180   L  CA      57.2287
  180   L  CB      41.6182
  180   L  C      179.4278
  180   L  H        8.4552
  180   L  HA       3.9733
  180   L  HB2      1.7245
  180   L  HB3      1.7019
  180   L  HD11     0.9191
  180   L  HD12     0.9190
  180   L  HD13     0.9193
  180   L  HD21     0.2312
  180   L  HD22     1.6504
  180   L  HD23     0.8685
  180   L  HG       1.7206
  181   A  N      120.7668
  181   A  CA      55.2218
  181   A  CB      18.3003
  181   A  C      179.6576
  181   A  H        7.9038
  181   A  HA       3.8984
  181   A  HB1      1.3861
  181   A  HB2      1.3625
  181   A  HB3      1.3317
  182   C  N      114.6661
  182   C  CA      60.1426
  182   C  CB      42.2188
  182   C  C      175.6220
  182   C  H        7.7173
  182   C  HA       4.0927
  182   C  HB2      3.0015
  182   C  HB3      3.2688
  183   L  N      120.5103
  183   L  CA      57.8498
  183   L  CB      40.9319
  183   L  C      179.5111
  183   L  H        8.1767
  183   L  HA       3.9876
  183   L  HB2      1.9326
  183   L  HB3      1.6990
  183   L  HD11     0.9195
  183   L  HD12     0.9939
  183   L  HD13     0.9045
  183   L  HD21     0.0133
  183   L  HD22     2.3852
  183   L  HD23     0.8711
  183   L  HG       1.6740
  184   S  N      112.3050
  184   S  CA      61.0863
  184   S  CB      62.3457
  184   S  C      175.8777
  184   S  H        7.9390
  184   S  HA       4.1095
  184   S  HB2      3.9933
  184   S  HB3      4.0030
  185   L  N      121.4553
  185   L  CA      57.2107
  185   L  CB      41.6189
  185   L  C      174.8820
  185   L  H        7.5898
  185   L  HA       4.1604
  185   L  HB2      1.6941
  185   L  HB3      1.6854
  185   L  HD11     0.9091
  185   L  HD12     0.9088
  185   L  HD13     0.9119
  185   L  HD21     0.3064
  185   L  HD22     2.5429
  185   L  HD23     0.8581
  185   L  HG       1.7972
  186   C  N      111.7128
  186   C  CA      54.4568
  186   C  CB      42.9806
  186   C  C      172.0526
  186   C  H        7.3902
  186   C  HA       5.0694
  186   C  HB2      3.3959
  186   C  HB3      3.2780
  187   H  N      117.0721
  187   H  CA      54.9981
  187   H  CB      28.4388
  187   H  C      176.0766
  187   H  H        8.2716
  187   H  HA       4.7125
  187   H  HB2      3.1274
  187   H  HB3      3.3104
  187   H  HD2      5.8800
  187   H  HE1      6.7350
  188   I  N      129.8203
  188   I  CA      60.2065
  188   I  CB      35.8495
  188   I  C      176.5377
  188   I  H        8.1285
  188   I  HA       4.0405
  188   I  HB       1.8760
  188   I  HD11     0.8595
  188   I  HD12     0.8601
  188   I  HD13     0.8562
  188   I  HG12     1.2747
  188   I  HG13     0.5257
  188   I  HG21     0.9157
  188   I  HG22     0.9151
  188   I  HG23     0.9133
 *189   E  N      119.2708
 *189   E  CA      55.8640
 *189   E  CB      30.2529
 *189   E  C      175.8263
 *189   E  H        8.2231
 *189   E  HA       4.1606
 *189   E  HB2      1.7441
 *189   E  HB3      1.7709
 *189   E  HG2      2.1296
 *189   E  HG3      2.3766
